packages/lib/src/index.ts

```diff
@@ -88,6 +101,23 @@ export interface CompletionOptions<TModel, TInput extends Input = Input> {
   numOutputs?: number
 }
 
+// MediaOptions allows you to specify options for the media generation request.
+export interface MediaOptions<TModel> {
+  // Identifier of the model to use. Defaults to the user's current model, but can be overridden here.
+  model?: TModel
+  // How many completion choices to attempt to generate. Defaults to 1. If the
+  // model doesn't support more than one, then an array with a single element will be returned.
+  numOutputs?: number
+  // type of media to generate
+  mimeType?: MediaMimeType 
```

Comment:
since PLY is currently hardcoded into the request params, can we remove this for now?

Reply:
Yes, let's remove. API's like openai's dalle & replicate's stable diffusion don't let you choose anyway.